### PR TITLE
Rename deploy method to capture metrics

### PR DESCRIPTION
**Pull request by sanjanasandeep**

### CODE REVIEW

**Review by de-lan — APPROVED** [review]

File: cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java [hunk 1]
```diff
@@ -187,7 +187,7 @@ public class AppLifecycleHttpHandler extends AbstractAppFabricHttpHandler {
   @PUT
   @Path("/apps/{app-id}")
   @AuditPolicy(AuditDetail.REQUEST_BODY)
-  public BodyConsumer deploy(HttpRequest request, HttpResponder responder,
+  public BodyConsumer create(HttpRequest request, HttpResponder responder,
```
Comment (de-lan) [inline cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java:190]: Nit: revert the comment change as well?

**Review by albertshau — APPROVED** [review]

File: cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java (hunk 1 above)
Comment (sanjanasandeep) [reply cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java:190]: the comment wasn't changed earlier?

Comment (de-lan) [reply cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java:190]: In line 185

**Review by de-lan — APPROVED** [review]

File: cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java (hunk 1 above)
Comment (sanjanasandeep) [reply cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java:190]: changed deploy to create

**Pull request merged**